### PR TITLE
New package: sile-0.9.4

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

File: srcpkgs/sile/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" lua51 lua51-lpeg luaexpat lua51-zlib luafilesystem"
+fi
+
+post_extract() {
+	sed 's/rm -f core/rm -f/' -i configure
+}
+
+do_install() {
```
Comment (newbluemoon) [inline srcpkgs/sile/template]: Typo? Did you mean `post_install()`? Else the created package just contains the license.

File: srcpkgs/sile/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'sile'
+pkgname=sile
+version=0.9.4
+revision=1
+_libtexpdf_commit="0cb0c20a3ba40057e6902551300630fed7199bf6"
```
Comment (maxice8) [inline srcpkgs/sile/template]: why is this here ?

Comment (Johnnynator) [reply srcpkgs/sile/template]: Because I missed to remove it. I had used the github generated tar ball first and the subrepo had to be fetched manually.

**Comment by abenson** [comment]
Closes wrong issue. https://github.com/voidlinux/void-packages/issues/14819

**Comment by Johnnynator** [comment]
The commit now references the correct issue.
